docs/dev-guide/README.md

```diff
@@ -0,0 +1,19 @@
+# Developer's Guide
+
+You can find the hosted dev-guide at
+
+https://gluwa.github.io/creditcoin/
+
+## Requirements
+```
+cargo install mdbook mdbook-mermaid
+```
+
+## Build & view
+```bash
+# to build book
+mdbook build
+
+# to host locally for immediate perusing
+mdbook serve
+```
```

Comment:
But even better would be for there to be a script that can be run both in CI and locally.

Depending on Github actions and pointing to inline scripts is sometimes confusing because they're usually conflated with CI-specific dependencies and deployment logic.

Pointing to the deploy-docs.yml requires the reader to hunt for the one line they need in the inline script:

 https://github.com/gluwa/creditcoin/blob/3815fd22f1a4b08cffb3a34a6dd47812f628a3d4/.github/workflows/deploy-docs.yml#L29

Reply:
Oh, I meant something like 

"Also see  `.github/workflows/deploy-docs.yml` for reference"

Reply:
done